app/src/componentLibrary/attributeEditor/attributeEditor.jsx

```diff
@@ -134,6 +134,16 @@ export const AttributeEditor = ({
   };
   const handleAttributeKeyInputChange = (text) => setState({ ...state, isKeyEdited: !!text });
 
+  const isValidForm = !isFormValid();
+
+  const handleKeyDown = (handler) => ({ keyCode }) => {
+    const enterKeyCode = 13;
```

Comment:
I see the list of Key code constants inside of the `app/src/componentLibrary/dropdown/constants.js`.
Seems like it's time to move them upper to the `common/constans/keyCodes.js` to use application-wide.